client/payment-details/summary/__tests__/index.test.js

```diff
@@ -139,14 +138,12 @@ const createTapToPayMetadata = ( readerModel, platform ) => ( {
 
 function renderCharge( charge, metadata = {}, isLoading = false, props = {} ) {
 	const { container } = render(
-		<WCPaySettingsContext.Provider value={ global.wcpaySettings }>
```

Comment:
I removed this wrapper because `PaymentDetailsSummary` is already wrapped by `WCPaySettingsContext.Provider`